server/sonar-server/src/main/java/org/sonar/server/platform/BackendCleanup.java

```diff
@@ -166,11 +171,29 @@ private static void truncateDefault(String tableName, Statement ddlStatement, Co
   }
 
   /**
+   * Admin membership on default organization must never be deleted
+   */
+  private static void truncateOrganizationsMembership(String tableName, Statement ddlStatement, Connection connection) throws SQLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "tableName". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)